messaginginapp/src/main/java/io/customer/messaginginapp/gist/presentation/GistSdk.kt

```diff
@@ -280,6 +341,7 @@ interface GistListener {
     fun embedMessage(message: Message, elementId: String)
     fun onMessageShown(message: Message)
     fun onMessageDismissed(message: Message)
+    fun onMessageCancelled(message: Message)
```

Comment:
Is this interface part of the public API for customers? 

I am OK adding this method if it's for internal code, only.

Reply:
It's `internal` only